azure-mgmt-compute/src/main/java/com/azure/management/compute/implementation/VirtualMachineScaleSetImpl.java

```diff
@@ -334,7 +334,7 @@ public Network getPrimaryNetwork() throws IOException {
     @Override
     public LoadBalancer getPrimaryInternetFacingLoadBalancer() throws IOException {
         if (this.primaryInternetFacingLoadBalancer == null) {
-            loadCurrentPrimaryLoadBalancersIfAvailable();
+            loadCurrentPrimaryLoadBalancersIfAvailable().block();
```

Comment:
Maybe you can name it as loadCurrentPrimaryLoadBalancersIfAvailableAsync to make it clearer that it is async method.